publications/models.py

```diff
@@ -19,12 +18,12 @@ class Publication(ElasticMixin, DorMixin, models.Model):
     sub_type = models.CharField(max_length=255, blank=True)
     title = models.TextField(blank=True)
     author = models.TextField(blank=True)
-    date = models.TextField(blank=True)
+    year = models.TextField(blank=True)
     abstract = models.TextField(blank=True)
     cite = models.TextField(blank=True)
     url = models.TextField(blank=True)
+    studies = models.TextField(blank=True)
```

Comment:
What is stored in this field? Can you give an example?

Reply:
This field contains a description of studies/datasets that the author used for the publication.

It will be documented in the data model.